docs/tutorials/lut.rst

```diff
@@ -25,78 +25,119 @@ We use here the WGS84 reference ellipsoid (used as standard by the Global Positi
+Bundling the grid specification
+###############################
+The grid specification needs to be bundled into a dictionary to be used as an input for the `compute_traveltimes` function.
 
 ::
 
-	from QMigrate.lut import compute, LUT, read_nlloc
-
-	# --- Create a new LUT ---
-	lut = LUT(ll_corner=ll_corner, ur_corner=ur_corner, cell_size=cell_size,
-	          grid_proj=gproj, coord_proj=cproj)
+    grid_spec = {"ll_corner": ll_corner,
+    			 "ur_corner": ur_corner,
+    			 "cell_size": cell_size,
+    			 "coord_proj": cproj,
+    			 "grid_proj": gproj}
 
 Computing traveltimes
 ---------------------
 We have bundled a few methods of computing traveltimes into QuakeMigrate.
 
-In all cases we will make use of the :mod:`QMigrate.io` module, so let's import that first and read in our station file:
+In addition to the grid specification, we need to provide a list of stations for which to compute traveltime tables.
 
 ::
 
-    from QMigrate.io import read_stations, read_vmodel
+    from QMigrate.io import read_stations
 
     stations = read_stations("/path/to/station_file")
 
+The `read_stations` function is a passthrough for `pandas.read_csv`, so we can handle any delimiting characters. There are four required (case-sensitive) column
+headers - "Name", "Longitude", "Latitude", "Elevation".
+
+.. warning:: Note: station elevations are in the positive-up/right-handed coordinate frame. An elevation of 2 would correspond to 2 (km) above sea level.
+
+The `compute_traveltimes` function used in the following sections returns a fully-populated instance of the LUT class.
+
 Homogeneous velocity model
 ##########################
-Simply calculates the straight line traveltimes between stations and points in the grid.
+Simply calculates the straight line traveltimes between stations and points in the grid. It is possible to use stations that are outside the specified span of the grid if you are able to constrain the possible sources of your seismicity (e.g. basal icequakes), which can greatly reduce the computational cost of running QuakeMigrate.
 
 ::
 
-	compute(lut, stations, method="homogeneous", vp=5000., vs=3000.)
+    from QMigrate.lut import compute_traveltimes
+
+	compute_traveltimes(grid_spec, stations, method="homogeneous", vp=5., vs=3.,
+	                    log=True, save_file=/path/to/save_file)
 
 Fast-marching method
 ####################
-The fast-marching method implicitly tracks the evolution of the wavefront. See Rawlinson & Sambridge (2005) for more details.
+The fast-marching method implicitly tracks the evolution of the wavefront. Our current backend is the `scikit-fmm` package. It is possible to use this package to compute traveltimes to 1-D, 2-D, or 3-D velocity models. Currently we provide a utility function that computes traveltime tables for 1-D velocity models. The format of this velocity model file is specified below. See the `scikit-fmm` documentation and Rawlinson & Sambridge (2005) for more details.
```

Comment:
```suggestion
The fast-marching method implicitly tracks the evolution of the wavefront. Our current backend is the `scikit-fmm` package. It is possible to use this package to compute traveltimes to 1-D, 2-D, or 3-D velocity models. Currently we provide a utility function that computes traveltime tables for 1-D velocity models. The format of this velocity model file is specified below. See the `scikit-fmm` documentation and Rawlinson & Sambridge (2005) for more details.

.. note:: Traveltime calculation can only be performed between grid nodes: the station location is therefore taken as the closest grid node. Note that for large cell sizes this may cause a modest error in the calculated traveltimes.

.. note:: All stations must be situated within the grid on which traveltimes are to be computed.

```